src/app/components/Cart/Cart.tsx

```diff
@@ -1,24 +1,86 @@
 import React from 'react';
-import { Box, Typography } from '@mui/material/';
-import CartItem from './CartItem/CartItem';
+import {
+  Avatar,
+  Box,
+  IconButton,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Typography,
+} from '@mui/material/';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useCartStore } from '../../../../lib/store/cart';
 
-const Cart: React.FC = () => {
-  const productsMock: {
-    id: number;
-    title: string;
-    price: number;
-    quantity: number;
-  }[] = [
-    { id: 1, title: 'Product 1', price: 10, quantity: 1 },
-    { id: 2, title: 'Product 2', price: 20, quantity: 1 },
-    { id: 3, title: 'Product 3', price: 30, quantity: 1 },
-  ];
+const styles = {
+  productsWrapper: (theme: any) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+    paddingTop: '30px',
+  }),
+  productItem: (theme: any) => ({
+    borderRadius: '12px',
+    backgroundColor: `${theme.palette.WHITE}`,
+  }),
+  productContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    paddingLeft: '30px',
+    maxWidth: '60%',
+    width: '60%',
+  },
+
+  price: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+
+  image: {
+    width: '50px',
+    height: '50px',
+  },
+  icon: (theme: any) => ({
+    color: `${theme.palette.DARK_BROWN}`,
+  }),
+  text: {
+    display: 'flex',
+    justifyContent: 'end',
+  },
+};
+
+const Cart = () => {
```

Comment:
jeśli tak dla Ciebie jest ok to spoko, ale nie wiem czy nie próbowałbym wykorzystać przykładu z dokumentacji : 
```
// Object pick, re-renders the component when either state.nuts or state.honey change
const { nuts, honey } = useBearStore(
  useShallow((state) => ({ nuts: state.nuts, honey: state.honey })),
)
``` 
i zrobienie hooka useCartState gdzie użycie będzie 
```
const { total, items, remove } = useCartState()